library/globals.inc.php

```diff
@@ -1588,6 +1588,13 @@
       xl('Strong password means at least 8 characters, and at least three of: a number, a lowercase letter, an uppercase letter, a special character.')
     ),
 
+    'secure_upload' => array(
```

Comment:
move this up to be above 'Require Strong Passwords' and below 'Idle Session Timeout Seconds'